lib/rspec/rails/fixture_support.rb

```diff
@@ -44,8 +56,6 @@ def name
               @example
```

Comment:
@Edouard-chin Do you mind to approve if I didn't break [your fix](https://github.com/rspec/rspec-rails/pull/2215)?

Reply:
Looks good to me 👍 . Thanks for the ping